keylime/mba/mba.py

```diff
@@ -185,3 +185,18 @@ def _find_implementation(functionname: str) -> Any:
         if hasattr(m, functionname) and isfunction(getattr(m, functionname)):
             return m
     raise ValueError(f"No implementation for function {functionname} found among measured boot imports")
+
+
+# ###########
+# Create a dictionary to write mb_policy into for mbpolicies table
+# ###########
+
+
+def mb_policy_db_contents(mb_policy_name: str, mb_policy: Optional[str]) -> Dict[str, Any]:
+    """Assembles a mb_policy dictionary to be written on the database"""
+    mb_policy_db_format: Dict[str, Any] = {}
+
+    mb_policy_db_format["name"] = mb_policy_name
+    mb_policy_db_format["mb_policy"] = mb_policy
+
+    return mb_policy_db_format
```

Comment:
This function seems to not really necessary.

Reply:
You are right. Somehow, i wrote this thinking about a generic fuction to return a db format for a given name and policy and policy could be verified in the same function before returing the format. But, we don't need that  at this time.

Reply:
This function is removed with the last commit.